metricbeat/module/kubernetes/apiserver/_meta/docs.asciidoc

```diff
@@ -1 +1,32 @@
-This is the `apiserver` metricset of the Kubernetes module.
+This is the `apiserver` metricset of the Kubernetes module, in charge of retrieving metrics from the Kubernetes API (available at `/metrics`).
+
+This metricset needs access to the `apiserver` component of Kubernetes, accessible typically by any POD via the `kubernetes.default` service or via environment variables (`KUBERNETES_SERVICE_HOST` and `KUBERNETES_SERVICE_PORT`).
+
+When the API uses https, the pod will need to authenticate using its default token and trust the server using the appropiate CA file. 
+
+Configuration example using https and token based authentication:
+
+```yaml
+    - module: kubernetes
+      enabled: true
+      metricsets:
+        - apiserver
+      hosts: ["https://${KUBERNETES_SERVICE_HOST}:${KUBERNETES_SERVICE_PORT}"]
+      #hosts: ["https://kubernetes.default"]
+      bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
+      ssl.certificate_authorities:
+        - /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+      period: 30s
+```
+
+In order to access the `/metrics` path of the API service, the following permission needs to be available by the service account running Metricbeat:
+
+```yaml
+rules:
+- nonResourceURLs:
+  - /metrics
+  verbs:
+  - get
+```
+
+The previous configuration and RBAC requirement is available in the complete example manifest proposed for `Running Metricbeat on Kubernetes` (link to main page or to the complete manifest?)
```

Comment:
++ on adding this link, it would be something like: `<<running-on-kubernetes,Running Metricbeat on Kubernetes>> `

Reply:
Perfect, but if we add it we need to make sure that the specific rule is added into the manifest, as the current one doesn't have it :)

Reply:
Marking this as resolved, I have added the link in the way you have suggested.